### PR TITLE
fix(sdk): optimized the tips for instant async quote

### PR DESCRIPTION
**Pull request by xuelianhan007**
## Related Issue

https://github.com/mycloudnexus/kraken/issues/25
## How This Has Been Tested
Integration test and unit test in localhost. This patch has worked as expected.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Refactoring or add test (improvements in base code or adds test coverage to functionality)
- [ ] CI/CD or documentation update (changes to CI/CD pipeline or documentation)

## Checklist

- [ ] I ensured that the documentation is up to date
- [ ] I explained why this PR updates in detail with reasoning why it's required
- [ ] I would like a code coverage CI quality gate exception and have explained why

### CODE REVIEW

**Review by kuangxiang20240501 — DISMISSED** [review]
LGTM

File: kraken-app/kraken-app-hub/src/test/resources/mock/mapping-matrix/mapping.matrix.quote.eline.add.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ spec:
       - name: enabled
         expectType: EXPECTED
         value: false
+        errorMsg: "instantSyncQuote==false"
```
Comment (zhiyaoPCCW) [inline kraken-app/kraken-app-hub/src/test/resources/mock/mapping-matrix/mapping.matrix.quote.eline.add.yaml:16]: errorMsg is a bit confused, should be a more reasonable message.

**Review by zhiyaoPCCW — COMMENTED** [review]
Should some improvements on errorMsg.

File: kraken-app/kraken-app-hub/src/test/resources/mock/mapping-matrix/mapping.matrix.quote.eline.add.yaml (hunk 1 above)
Comment (xuelianhan007) [reply kraken-app/kraken-app-hub/src/test/resources/mock/mapping-matrix/mapping.matrix.quote.eline.add.yaml]: Thanks. The tips have been redefined.

**Review by kuangxiang20240501 — APPROVED** [review]
LGTM

**Review by zhiyaoPCCW — APPROVED** [review]
LGTM

**Review by kuangxiang20240501 — APPROVED** [review]

**Pull request merged**